src/Components/Header/Header.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import * as ROUTES from '../../constants/routes';
+import Image from '../../pages/Login/Image/Image';
+import SvgComponent from './SvgComponent/SvgComponent';
+import PathComponent from './PathComponent/PathComponent';
+
+const Logo = {
+  src: '/images/logo.png',
+  alt: 'Instagram',
+  imageWrapperStyle: 'mt-2 w-6/12'
+};
+
+const SvgProps = {
+  className: 'w-8 mr-6 text-black-light cursor-pointer',
+  xmlns: 'http://www.w3.org/2000/svg',
+  fill: 'none',
+  viewBox: '0 0 24 24',
+  stroke: 'currentColor'
+};
+
+const PathProps1 = {
+  strokeLinecap: 'round',
+  strokeLinejoin: 'round',
+  strokeWidth: 2,
+  d: 'M3 12l2-2m0 0l7-7 7 7M5 10v10a1 1 0 001 1h3m10-11l2 2m-2-2v10a1 1 0 01-1 1h-3m-6 0a1 1 0 001-1v-4a1 1 0 011-1h2a1 1 0 011 1v4a1 1 0 001 1m-6 0h6'
+};
+
+const PathProps2 = {
```

Comment:
Could you please rename it to "logoutIconProps"? 
PathProps2 does not say what it actually is, and the code will not be clean.